### PR TITLE
chore: JSON.GET now works with our own jsonpath

### PR DESCRIPTION
**Pull request by romange**

### CODE REVIEW

File: src/core/json/jsonpath_test.cc [hunk 1]
```diff
@@ -30,6 +30,12 @@ class TestDriver : public Driver {
   }
 };
 
+inline JsonType ValidJson(string_view str) {
```
Comment (romange) [inline src/core/json/jsonpath_test.cc:33]: a convenience wrapper

File: src/core/json/jsonpath_test.cc [hunk 2]
```diff
@@ -123,6 +129,19 @@ TEST_F(JsonPathTest, Parser) {
   EXPECT_EQ(0, Parse("$.plays[*].game"));
 }
 
+TEST_F(JsonPathTest, Root) {
```
Comment (romange) [inline src/core/json/jsonpath_test.cc:132]: the only change is this test. Now for empty Path we return the whole object to satisfy `$` query.

File: src/server/json_family.cc [hunk 3]
```diff
@@ -65,6 +65,17 @@ inline void Evaluate(const json::Path& expr, const JsonType& obj, ExprCallback c
   });
 }
 
+inline JsonType Evaluate(const JsonExpression& expr, const JsonType& obj) {
```
Comment (romange) [inline src/server/json_family.cc:68]: a special variant of Evaluate that does not accept a callback and instead returns all the hits as a json array.

File: src/server/json_family.cc [hunk 4]
```diff
@@ -65,6 +65,17 @@ inline void Evaluate(const json::Path& expr, const JsonType& obj, ExprCallback c
...
+  return expr.evaluate(obj);
+}
+
+inline JsonType Evaluate(const json::Path& expr, const JsonType& obj) {
```
Comment (romange) [inline src/server/json_family.cc:72]: same variant for `json::Path`

**Review by BorysTheDev — APPROVED** [review]

**Pull request merged**